frontend/src/pages/user-profile/components/user-profile-sidebar/styles.module.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+  width: 350px;
+  height: 100%;
+  background: linear-gradient(
+    to bottom,
+    var(--transparent) 0,
+    var(--blue-100) 100%
+  );
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 10%;
+  padding: 0 20px;
+  font-weight: 600;
+  font-size: 20px;
+  border-bottom: 1px solid var(--blue-100);
+}
+
+.avatar-container {
+  width: 72px;
+  height: 72px;
```

Comment:
same about width and height, if it's related to image then add it via attributes

Reply:
done